examples/admin_remove_user/admin_remove_user.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"context"
+	"examples"
+	"fmt"
+	"os"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+)
+
+func main() {
+	defer examples.HandlePanic()
+
+	const server = "http://localhost:3003/v1"
+
+	if examples.ExampleAdminKey == "" {
+		fmt.Println(examples.ErrMessage("adminKey"))
+		os.Exit(1)
+	}
+
+	adminClient := walletclient.NewWithAdminKey(server, examples.ExampleAdminKey)
+	ctx := context.Background()
+
+	adminClient.AdminDeletePaymail(ctx, examples.ExamplePaymail)
```

Comment:
This function returns error - we should get it and handle correctly. Just to show an user of this package good practices.

Reply:
done